api/pkg/imagebuilder/imagebuilder.go

```diff
@@ -474,22 +479,27 @@ func (c *imageBuilder) createKanikoJobSpec(project mlp.Project, model *models.Mo
 		}
 	}
 
-	resourceRequirements := RequestLimitResources{
+	var resourceRequirements RequestLimitResources
+	cpuRequest := resource.MustParse(c.config.DefaultResourceRequest.CPURequest)
+	memoryRequest := resource.MustParse(c.config.DefaultResourceRequest.MemoryRequest)
+
+	if resourceRequest != nil {
+		if !resourceRequest.CPURequest.IsZero() {
+			cpuRequest = resourceRequest.CPURequest
+		}
+		if !resourceRequest.MemoryRequest.IsZero() {
+			memoryRequest = resourceRequest.MemoryRequest
+		}
+	}
+
+	resourceRequirements = RequestLimitResources{
 		Request: Resource{
-			CPU: resource.MustParse(
-				c.config.Resources.Requests.CPU,
-			),
-			Memory: resource.MustParse(
-				c.config.Resources.Requests.Memory,
-			),
+			CPU:    cpuRequest,
+			Memory: memoryRequest,
 		},
 		Limit: Resource{
-			CPU: resource.MustParse(
-				c.config.Resources.Limits.CPU,
-			),
-			Memory: resource.MustParse(
-				c.config.Resources.Limits.Memory,
-			),
+			CPU:    cpuRequest,
```

Comment:
can we keep the previous platform behaviour?
```
cpuRequest := resource.MustParse(c.config.Resources.Requests.CPU)
	memoryRequest := 
cpuRequest := resource.MustParse(c.config.Resources.Requests.Memory)
	memoryRequest := 
cpuLimit := resource.MustParse(c.config.Resources.Limits.CPU)
	memoryRequest := 
MemoryLimit:= resource.MustParse(c.config.Resources.Limits.Memory)
```
only if `if resourceRequest != nil`, we set request == limit